app/src/Blocks/Column/Item.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace SilverStripe\Bambusa\Blocks\Column;
+
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
+use SilverStripe\Forms\RequiredFields;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Versioned\Versioned;
+
+class Item extends DataObject
```

Comment:
There's a bit of duplicate code in here, see https://github.com/dnadesign/silverstripe-elemental/blob/master/src/Models/ElementalArea.php

It may be worth looking into working from https://github.com/dnadesign/silverstripe-elemental-list if possible.

Reply:
> There's a bit of duplicate code in here

Agree, though it's still different. E.g. it doesn't have a title and there are other discrepancies which in general makes it easier not to reuse.

> It may be worth looking into working from https://github.com/dnadesign/silverstripe-elemental-list if possible

Yes, I assessed that module and decided it's not worth adding it as an extra dependency because we won't be able to reuse most of it anyway. On the other hand, most of the reusable parts we could take from it we would have to override anyway, which would increase complexity without giving much benefits in return.